acesso-io/acessobio-ios
# refactor ErrorHttp class according @obj protocol

Why?
To allow ErrorHttp class visibility in the ObjC context

Changes
Move ErrorHttp class from struct to nsobject 

How to test
xcodebuild run test

Preview
n/a

Issues
https://github.com/acesso-io/squad-check-mobile/issues/274
